employee-rostering-backend/pom.xml

```diff
@@ -171,6 +175,23 @@
       <artifactId>optaplanner-test</artifactId>
       <scope>test</scope>
     </dependency>
+    
+    <!-- Logging -->
+   <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+    </dependency>
+    
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jcl-over-slf4j</artifactId>
```

Comment:
This doesn't seem to be needed either.

Reply:
I got an error relating to commons-logging when I excluded the dependency, so this add the needed dependencies.